config/jail.conf

```diff
@@ -767,3 +767,15 @@ port = 2222
 enabled  = false
 logpath  = /var/lib/portsentry/portsentry.history
 maxretry = 1
+
+[pass2allow-ftp]
+# this pass2allow example allows FTP traffic after successful HTTP authentication
+filter     = apache-pass
+blocktype  = RETURN
+returntype = DROP
+# access log of the website with HTTP auth
```

Comment:
I think we need to spell out the action here and explicitly state that "knocking_url" needs to be customized per each deployment, so people do not stick with common default, allowing attacker to try it hoping that fail2ban is forbidding them access due to need to knock